<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>866</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="loginEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#loginEdit {
border: none;
border-bottom: 2px solid rgb(0, 32, 148);
border-top: 2px solid rgb(0, 32, 148);
background-color: rgba(9, 9, 9, 0);
color: rgba(255, 255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#passEdit {
border: none;
border-bottom: 2px solid rgb(0, 32, 148);
border-top: 2px solid rgb(0, 32, 148);
background-color: rgba(9, 9, 9, 0);
color: rgba(255, 255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>540</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>351</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#check_btn {
color: rgb(255, 255, 255);
border: 2px Solid rgb(255, 255, 255);
background-color: none
}
QPushButton#check_btn:hover {
color: rgb(255, 255, 255);
border: 5px Solid rgb(255, 255, 255);
background-color: none;
}</string>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QLabel" name="Error_lbl">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>750</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>610</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#delete_btn {
color: rgb(255, 255, 255);
border: 2px Solid rgb(255, 255, 255);
background-color: none
}
QPushButton#delete_btn:hover {
color: rgb(255, 255, 255);
border: 5px Solid rgb(255, 255, 255);
background-color: none;
}</string>
   </property>
   <property name="text">
    <string>Удалить пользователя</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>471</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/sea.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Error_lbl_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Авторизация</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>loginEdit</zorder>
  <zorder>passEdit</zorder>
  <zorder>check_btn</zorder>
  <zorder>Error_lbl</zorder>
  <zorder>delete_btn</zorder>
  <zorder>Error_lbl_2</zorder>
 </widget>
 <resources>
  <include location="../../../../Downloads/background.qrc"/>
 </resources>
 <connections/>
</ui>
